taskcluster/ci/lint/kind.yml

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - taskgraph.transforms.job:transforms
+  - taskgraph.transforms.task:transforms
+
+job-defaults:
+  worker-type: b-linux
+  worker:
+    docker-image: {in-tree: linux}
+    max-run-time: 1800
+  run:
+    using: run-task
+    cwd: '{checkout}'
+
+jobs:
+  detekt:
+    description: 'detekt'
+    run:
+      command: ./gradlew --no-daemon clean detekt
```

Comment:
Followup: I should have had noted this on the first pass. Let's use the `gradlew` job type, just like fenix does:
 * Usage: https://github.com/mozilla-mobile/fenix/blob/f8048462b584d04998aec3a3d513e5c6b57697f8/taskcluster/ci/lint/kind.yml#L54-L56
 * Job type definition: https://github.com/mozilla-mobile/fenix/blob/f8048462b584d04998aec3a3d513e5c6b57697f8/taskcluster/fenix_taskgraph/job.py#L70-L82

Reply:
Addressed in https://github.com/mozilla/application-services/pull/3113